packages/webassemblyjs/test/fixtures/br/desugared-spec-test/exec.tjs

```diff
@@ -4,20 +4,20 @@
 it("should branch correctly", () => {
   const { exports } = WebAssembly.instantiateFromSource(watmodule);
 
-  assert.equal(exports['type-i32'](), undefined);
+  assert.equal(exports["type-i32"](), undefined);
   // assert.equal(exports['type-i64'](), undefined);
-  assert.equal(exports['type-f32'](), undefined);
-  assert.equal(exports['type-f64'](), undefined);
+  assert.equal(exports["type-f32"](), undefined);
+  assert.equal(exports["type-f64"](), undefined);
 
-  assert.equal(exports['type-i32-value'](), 1);
+  assert.equal(exports["type-i32-value"](), 1);
   // assert.equal(exports['type-i64-value'](), 2);
-  assert.equal(exports['type-f32-value'](), 3);
-  assert.equal(exports['type-f64-value'](), 4);
+  assert.equal(exports["type-f32-value"](), 3);
+  assert.equal(exports["type-f64-value"](), 4);
 
-  assert.equal(exports['as-block-first'](), undefined);
-  assert.equal(exports['as-block-mid'](), undefined);
-  assert.equal(exports['as-block-last'](), undefined);
-  assert.equal(exports['as-block-value'](), 2);
+  assert.equal(exports["as-block-first"](), undefined);
+  assert.equal(exports["as-block-mid"](), undefined);
+  assert.equal(exports["as-block-last"](), undefined);
+  assert.equal(exports["as-block-value"](), 2);
 
   assert.equal(exports["as-loop-first"](), 3);
   assert.equal(exports["as-loop-mid"](), 4);
```

Comment:
Oops I didnt mean to edit this, just ran `make fix`?

Reply:
That's fine, the linter doesn't pass on `.tjs` IRC but the fix does.